utils/speedtest.py

```diff
@@ -0,0 +1,110 @@
+from io import BytesIO
+from numpy import random
```

Comment:
Please use the in-build random module (https://docs.python.org/3.5/library/random.html). `numpy` is a huge dependency which is not really needed here.

Reply:
1. numpy is needed for tests and utils only (for developers). The whole library does not depend on it.

2. I started from built-in library. In case you need to fill 10 MB byte buffer with random data it becomes extra slow (like 3 MB/s). Using existing call in numpy works at 500 MB/s

Reply:
How about `os.urandom`? It's built-in and seems to be quite fast - https://stackoverflow.com/a/7044536/179729 .